docs/CircleCI.md

```diff
@@ -1,72 +1,90 @@
-
-Additionally, an approach to assist in configuring reusable pieces of configuration within a Buildkite pipeline is through the use of [YAML anchors](https://buildkite.com/docs/plugins/using#using-yaml-anchors-with-plugins) - which are defined and used in a similar way to CircleCI `aliases`. 
\ No newline at end of file
+| Key | Supported? | Notes |
+| --- | --- | --- |
+| `jobs` | ✅ | A collection of steps that are run on a single worker unit: whether that is on a host directly, or on a virtualised host (such as within a Docker container). Orchestrated with `workflows`. |
+| `jobs.<name>` | ✅ | The named, induvidual `jobs` that makes up part of a given `workflow`. |
+| `jobs.<name>.environment` | ✅ | The `job` level environment variables of a CircleCI pipeline. Applied in the generated [command step](https://buildkite.com/docs/pipelines/command-step) as [step level](https://buildkite.com/docs/pipelines/environment-variables#runtime-variable-interpolation) environment variables with the `env` key. |
+| `jobs.<name>.parallelism` | ❌ | A `parallelism` parameter (if defined greater than 1) will create a seperate execution environment and will run the `steps` of the specific `job` in parallel. In Buildkite - a similar `parallelism` key can be set to a [command step](https://buildkite.com/docs/tutorials/parallel-builds#parallel-jobs) which will run the defined `command` over seperate jobs (sharing the same agent [queues](https://buildkite.com/docs/agent/v3/queues#setting-an-agents-queue)/[tags](https://buildkite.com/docs/agent/v3/cli-start#setting-tags) targets). |
+| `jobs.<name>.parameters` | ✅ | Reusable keys that are used within `step` definitions within a `job`. Default parameters that are specified in a `parameters` block are passed through into the [command step](https://buildkite.com/docs/pipelines/command-step)'s `commands` if specified. |
+| `jobs.<name>.shell` | ❌ | The `shell` property sets the default shell that is used across all commands within all steps. This should be configured on the agent - or by defining the `shell` [option](https://buildkite.com/docs/agent/v3/cli-start#shell) when starting a Buildkite agent: which will set the shell command used to interpret all build commands. |
+| `jobs.<name>.steps` | ✅ | A collection of non-`orb` `jobs` commands that are executed as part of a CircleCI `job`. Steps can be defined within an `alias`. All `steps` within a singular `job` are translated to the `commands` of a shared [command step](https://buildkite.com/docs/pipelines/command-step) within the generated Buildkite pipeline to ensure they share the same execution environment. |
+| `jobs.<name>.working_directory` | ✅ | The location of the executor where steps are run in. If set, a change directory (`cd`) command is created within the shared `commands` of a Buildkite [command step](https://buildkite.com/docs/pipelines/command-step) to the desired location. |
+
+#### Executors 
+
+| Key | Supported? | Notes |
```

Comment:
I would add a note here stating that, while the translation sets up the correct targeting, for them to work they need to setup their agents with the necessary dependencies and tools (maybe using the AWS elastic stack).

Reply:
Just added in 👍